README.md

```diff
@@ -344,6 +344,8 @@ or to the latest git commit's timestamp with:
 export SOURCE_DATE_EPOCH=$(git log -1 --format='%ct')
 ```
 
+The same applies to `KO_DATA_DATE_EPOCH` which sets the last modified time of all files in `kodata`.
```

Comment:
Can you explain a little more why this is useful (e.g. mention modtime for https://golang.org/pkg/net/http/#ServeFile)?

Reply:
**I improved the docs a bit.**
As for examples for the timestamp source I could not think of anything useful.
Maybe I could add something more complex like https://stackoverflow.com/questions/4561895/how-to-recursively-find-the-latest-modified-file-in-a-directory.

Let me know what you think.